src/Bindings/DeprecatedBindings.cpp

```diff
@@ -30,7 +30,7 @@ static int tolua_get_AllToLua_g_BlockLightValue(lua_State* tolua_S)
 	}
 	#endif
 	BlockType = static_cast<BLOCKTYPE>(tolua_tonumber(tolua_S, 2, 0));
-	if ((BlockType < 0) || (BlockType > E_BLOCK_MAX_TYPE_ID))
+	if (BlockType > E_BLOCK_MAX_TYPE_ID)
```

Comment:
because BLOCKTYPE is unsigned so comparisons against it being lesser than 0 are always true